doc/user/content/releases/v0.69.md

```diff
@@ -2,7 +2,7 @@
 title: "Materialize v0.69"
 date: 2023-09-20
 released: true
-patch: 2
+patch: 5
```

Comment:
This has not kept up with reality, causing the Nightly of this PR to hit the bug that was fixed in v0.69.5